demos/twa-play-billing/build.gradle

```
@@ -0,0 +1,49 @@
+/*
+ *    Copyright 2019 Google LLC
```

nit: 2020. I think all headers currently read 2019.

```suggestion
 *    Copyright 2020 Google LLC
```